src/app.tsx

```diff
@@ -34,28 +34,39 @@ import { NavigatorCardBody } from "./navigator-card-body.jsx";
 import { SidebarPanelDetails } from "./sidebar.jsx";
 import { NavigatorCardHeader } from "./header.jsx";
 import { usePageLocation } from "hooks.js";
-import { fsinfo } from "./fsinfo";
+import { fsinfo, FileInfo } from "./fsinfo";
 
 superuser.reload_page_on_change();
 
+interface Alert {
+    key: string,
+    title: string,
+    variant: AlertVariant,
+}
+
+interface NavigatorFileInfo extends FileInfo {
```

Comment:
Will be nice if we can get rid of this some day, but in the present, this is the best possible description of this thing.  :+1: